lib/ts/fetch.ts

```diff
@@ -428,22 +433,24 @@ export async function onUnauthorisedResponse(
                 );
 
                 logDebugMessage("onUnauthorisedResponse: Refresh call ended");
-                let removeIdRefreshToken = true;
-                const idRefreshToken = response.headers.get("id-refresh-token");
-                if (idRefreshToken) {
-                    logDebugMessage(
-                        "onUnauthorisedResponse: Setting sIRTFrontend from refresh API call: " + idRefreshToken
-                    );
-                    await setIdRefreshToken(idRefreshToken, response.status);
-                    removeIdRefreshToken = false;
-                }
-                if (response.status === AuthHttpRequest.config.sessionExpiredStatusCode) {
-                    logDebugMessage("onUnauthorisedResponse: Refresh status code is: " + response.status);
-                    // there is a case where frontend still has id refresh token, but backend doesn't get it. In this event, session expired error will be thrown and the frontend should remove this token
-                    if (removeIdRefreshToken) {
-                        await setIdRefreshToken("remove", response.status);
-                    }
+
+                await saveTokensFromHeaders(response);
```

Comment:
earlier, we used to save front-token and ant-csrf token only if status code was < 300. But now we do it even if status code is >=300. Are there any side effects?